task1.java

```diff
@@ -0,0 +1,34 @@
+package com.Selenium.homework1;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class task1 {
+    public static void main(String[] args) {
+
+        List<String> l1 = new LinkedList<String>();
+        l1.add("car");
+        l1.add("bus");
+        l1.add("taxi");
+        l1.add("taxi");
+        l1.add("bus");
+        l1.add("bus");
+        l1.add("train");
+        l1.add("taxi");
+
+        //System.out.print("\t" + l1);
+        List<String> head = l1.subList(0, 4);
+        List<String> tail = l1.subList(5, 8);

```

Comment:
smart
but it's not flexible, because we use real numbers
what if there will be other length of the list? will you update that numbers each time?
try to manipulate with l1.size()
you can use something like `l1.subList(0, l1.size()/2)`

Reply:
btw why did you specify tail if you didn't used that?

Reply:
fixed in 'task1.changes'